### PR TITLE
MicroManagerReader: Minor fix to handle metadata block closing

### PR DESCRIPTION
**Pull request by bramalingam**
ref: https://trello.com/c/dqTlLuXf/637-serge-pelet-single-cell-data#comment-5a783cf80e00f887aa369c3f

Testing Instructions:
1) Test via Fiji, Command line tools and check if the file opens appropriately via Bio-Formats.
Without this PR you should get an exception (as shown in the trello card).
2) Check if the subset job listed below is green.

@sbesson

### CODE REVIEW

**Comment by bramalingam** [comment]
Have triggered the merge_repository_subset job for the Micromanager files:

https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-merge-repository-subset/538/

File: components/formats-bsd/src/loci/formats/in/MicromanagerReader.java [hunk 1]
```diff
@@ -781,7 +781,6 @@ else if (key.equals("IJType")) {
         StringBuilder valueBuffer = new StringBuilder();
         boolean valueArray = false;
         int nestedCount = 0;
-
```
Comment (sbesson) [inline components/formats-bsd/src/loci/formats/in/MicromanagerReader.java]: Why this extra line?

**Comment by bramalingam** [comment]
Handling File Name redundancy : Job Triggered for the latest commit  : https://ci.openmicroscopy.org/job/BIOFORMATS-DEV-merge-repository-subset/539/

**Comment by sbesson** [comment]
As discussed with @bramalingam, I would separate the last commits trying to improve the `ImageName` parsing out of this PR. The bug fix allowing to open this type of Micro-Manager datasets should be largely ready to go into 5.8.0 while the former might require more back-n-forth.

**Comment by bramalingam** [comment]
--rebased-to #3071

**Pull request merged**